.github/workflows/build.yml

```diff
@@ -86,6 +86,40 @@ jobs:
         with:
           flags: dotnet
 
+  build-bicep-linux-musl-x64:
```

Comment:
Why not add it to the job matrix?

Reply:
Why do you want to build bicep on alpine linux? It's sufficient to build on any other system with rid set to musl.

I see that e2e tests that are using bicep executable will not work, but perhaps that step only could be containerised (`uses` directive in step, see: https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idstepsuses)

Reply:
Running the E2E tests requires `node` and `npm` installed. If using the `microsoft/azure-cli:latest` image we need to: install `node` and `npm` via apk, run `npm ci`, and run `npm test`. I could not figure out a way to do the three commands all in one containerized step since it only takes one `entrypoint`. Running the commands in multiple steps doesn't work either based on my tests. Do you mind giving me some pointers if you happen to know the proper setup?

Would like to know @majastrz's thoughts as well.